layouts/partials/head.html

```diff
@@ -33,8 +33,8 @@
   integrity="sha384-vRQ9bDyE0Wnu+lMfm57BlYLO0/XauFuKpVsZPs7KEDwYKktWi5+Kz3MP8++DFlRY"
   crossorigin="anonymous"></script>
 {{end}}
-{{ with .Site.Params.prism_syntax_highlighting }}
+{{ if .Site.Params.prism_syntax_highlighting }}
 <!-- stylesheet for Prism -->
-<link rel="stylesheet" href="/css/prism.css" />
+<link rel="stylesheet" href="{{ .Site.BaseURL }}/css/prism.css" />
```

Comment:
A somewhat "cleaner" way of creating this url (and the js one) is to use 
```html
<link rel="stylesheet" href="{{ "/css/prism.css" | relURL }}" />
```

That will correctly prepend `BaseURL` as needed

Reply:
Great point, thanks!